src/models/PartialFormSubmission.php

```diff
@@ -233,4 +233,17 @@ public function generateKey($token)
     {
         return hash_pbkdf2('sha256', $token, $this->TokenSalt, 1000, 16);
     }
+
+    /**
+     * Get all partial fields for loading data into the form
+     *
+     * @return array
+     */
+    public function getFields()
+    {
+        $formFields = $this->PartialFields()->map('Name', 'Value')->toArray();
+        $fileFields = $this->PartialUploads()->map('Name', 'Filename')->toArray();
+
+        return $formFields + $fileFields;
```

Comment:
Could you use array_merge here? I'm not a fan of using `+`, it works, but is unclear in its intention